src/k8s/cmd/k8s/k8s_get.go

```diff
@@ -27,13 +27,16 @@ func newGetCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 				return
 			}
 
-			config, err := client.GetClusterConfig(cmd.Context(), api.GetClusterConfigRequest{})
+			config, err := client.GetClusterConfig(cmd.Context(), apiv1.GetClusterConfigRequest{})
 			if err != nil {
 				cmd.PrintErrf("Error: Failed to get the current cluster configuration.\n\nThe error was: %v\n", err)
 				env.Exit(1)
 				return
 			}
 
+			// ignore MetricsServer config
```

Comment:
nit: this commment doesn't really add any context. IMHO, either remove it or add some context on *why* this is ignored.